### ISSUE TITLE
Custom font issues

### ISSUE THREAD

**Issue by akaraul** [andyfinnell/NativeMarkKit#5]
Hi @andyfinnell. I have a problem, when I set a custom font, italic and bold stop working.
```swift
    let style = StyleSheet.default.duplicate().mutate(
        block: [
            .document: [
//                .textStyle(.body),
                .textStyle(.custom(name: .custom("Avenir-Medium"), size: .fixed(18))),
                .backgroundColor(.white),
                .textColor(.black)
            ],
        ]
```

With custom font:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/20819174/115269936-7a802a80-a144-11eb-9226-9729e6a8f76e.png">

With system font:
<img width="276" alt="image" src="https://user-images.githubusercontent.com/20819174/115270022-9388db80-a144-11eb-9367-0d81b1a1d9c7.png">

**Comment by andyfinnell** [comment]
Hi @akaraul, thanks for the report. I am able to reproduce it. I'll work on a fix.

**Comment by andyfinnell** [comment]
@akaraul This should be fixed in the release tagged as `2.0.0`.

**Comment by akaraul** [comment]
@andyfinnell Now it works. Thanks)
